### ISSUE TITLE
PyTorch Cuda Tensors not supported in DETR model

### ISSUE THREAD

**Issue by dennisushi** [huggingface/transformers#15574]
https://github.com/huggingface/transformers/blob/db7d6a80e82d66127b2a44b6e3382969fdc8b207/src/transformers/models/detr/feature_extraction_detr.py#L568

The feature extractor attempts to convert the tensors to PIL images, which is only possible for `t.detach().cpu()` tensors. 

The origin of the error is `ImageFeatureExtractorMixin.to_pil_image()`

**Comment by LunaJang** [comment]
I'm having same problem. Did you solve it?

**Comment by LunaJang** [comment]
> I'm having same problem here. Did you solve it?
